packages/@lwc/perf-benchmarks-components/src/benchmark/tree/tree.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2024, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+import { LightningElement, api } from 'lwc';
+
+export default class extends LightningElement {
+    @api data = { id: 'foo' };
+    @api depth = 0;
+    @api breadth = 0;
+    @api index = 0;
+
+    get depthMinusOne() {
+        return this.depth - 1;
+    }
+
+    get notLeaf() {
+        return this.depth > 0;
+    }
+
+    get children() {
+        return Array(this.breadth)
+            .fill()
+            .map((_, i) => i);
```

Comment:
```suggestion
        return Array.from({ length: this.breadth }, (_, i) => i);
```
Dunno whether this is more perfy, but it's shorter!